modules/install-osp-dualstack.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+:_content-type: CONCEPT
+[id="install-osp-dualstack_{context}"]
+= Optional: Configuring a cluster with dual-stack networking
+
+:FeatureName: Dual-stack configuration for OpenStack
+include::snippets/technology-preview.adoc[]
+
+You can create a dual-stack cluster on {rh-openstack}. However, this is enabled only if you are using an {rh-openstack} network with IPv4 and IPv6 subnets. To configure a dual-stack cluster, you must complete the following tasks:
```

Comment:
```suggestion
You can create a dual-stack cluster on {rh-openstack}. However, this [what?] is enabled only if you are using an {rh-openstack} network with IPv4 and IPv6 subnets. 

To configure a dual-stack cluster, you must complete the following tasks:
```
- The pronoun `this` needs to have a clear reference -- 'this' what? See the ISG for more info: https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references
- Consider adding a paragraph break if you keep this all as a single concept module. I think I would probably break out this sentence and the following content into a separate procedure module.